tools/autograd/derivatives.yaml

```diff
@@ -1562,9 +1568,10 @@
   grad_output: avg_pool3d(grad, kernel_size, stride, padding, ceil_mode, count_include_pad, divisor_override)
   self: zeros_like(self)
 
-- name: elu_backward(Tensor grad_output, Scalar alpha, Scalar scale, Scalar input_scale, Tensor output) -> Tensor
-  grad_output: elu_backward(grad, alpha, scale, input_scale, output)
-  output: grad * grad_output * input_scale * (output < 0).type_as(grad)
+- name: elu_backward(Tensor grad_output, Scalar alpha, Scalar scale, Scalar input_scale, bool is_result, Tensor self) -> Tensor
```

Comment:
@albanD, I'm unsure how to update the derivative for the double backward case. What is the difference between grad, grad_output, and self for these formulas?

Reply:
grad_output is the grad from the first backward (renamed here to avoid name conflict with the grad from this op)
grad below is always the gradient flowing back for that op.
And self here is `self_or_result` that you give in the formula above.